workers.py

```diff
@@ -45,13 +45,19 @@ def _retryable(method: str, *args, **kwargs) -> requests.Response:
     raise requests.HTTPError('All attempts failed')
 
 
-def download_job(source_url: str, source_id: str, dest_url: str) -> None:
+def download_job(
+        source_url: str,
+        source_id: str,
+        dest_url: str,
+        b64_identity: str
+) -> None:
     """Spawn a thread worker for data downloading task.
 
     Requests the data to be downloaded and pass it to the next service
     :param source_url: Data source location
     :param source_id: Data identifier
     :param dest_url: Location where the collected data should be received
+    :param b64_identity: Identity in the request header
```

Comment:
Something along these lines fits better as a description:
```suggestion
    :param b64_identity: RedHat identity base64 string
```

Remember, this docstring is for a worker, it basically doesn't know anything about any requests and headers what so ever.

Reply:
Ok, done!